src/shared/components/case-viewer/case-viewer.component.ts

```diff
@@ -181,7 +172,7 @@ export class CaseViewerComponent implements OnInit, OnDestroy, AfterViewInit {
   }
 
   public hasTabsPresent(): boolean {
-    return this.sortedTabs.length > 0;
+    return this.sortedTabs.length > 0 || this.prependedTabs.length > 0;
```

Comment:
A few points to consider:
- the tabs need a feature toggle flag, only when it's toggle on and then these two addtional tabs can be shown, as we will release the code LIVE but the feature will be off at the first day.
- the tabs need permission check, only when user has legal ops and judicial user role then he can see the two additional tabs.
- Is the tabs configured in APP config and how?

Reply:
Hi Ray
The tabs will be configured from MC. Toolkit responsibility is to render the tabs, hence toolkit should have no knowledge of RBAC.

Reply:
this changes also need to cover the AC in:
https://tools.hmcts.net/jira/browse/EUI-3570

@udaydenduluri33 @kuda1992

Reply:
@rayliangatos This been moved to MC.